docs/scripts/buildApi.js

```diff
@@ -64,10 +77,6 @@ function buildDocs(options) {
   reactAPI.pagesMarkdown = pagesMarkdown;
   reactAPI.src = src;
 
-  // if (reactAPI.name !== 'Backdrop') {
```

Comment:
This comment is very useful for debugging, more than useful.

Reply:
Comment? Looks like commented out dead code.